.lycheeignore

```diff
@@ -1,6 +1,3 @@
-https://github.com/eclipse-iceoryx/iceoryx/issues/.*
-https://github.com/eclipse-iceoryx/iceoryx/pull/.*
```

Comment:
It seems this results in many link checker network errors. Maybe it's better to revert

Reply:
No need for that anymore with a github token and a max_concurrency of 1 it runs smoothly.